Updated paper with example images

voetberg opened the pull request:
Addressing issues #7 #10 #9 

Pulled the paper from the overleaf - let me know if there are more updates that need to be made.

bnord reviewed (changes requested):
We should do the following
1. update the images
    2. higher resolution for the overview_diagram
    3. remove the png's that don't have names
    4. update the grid of images
    5. add the physics plots
2. update the paper.md with the latest text from the overleaf

voetberg commented:
@bnord Addressed those issues. 

Only thing standing is that the compiled pdf from the make_pdf workflow does not have the citations, and the format of those citations is..odd. Is that something we should be concerned with, or is that more an issue with the action not being a perfect replication the joss submission workflow? I can't find anything wrong with either the workflow or the paper format so 🤷
